kuma/wiki/views/translate.py

```diff
@@ -214,7 +217,24 @@ def translate(request, document_slug, document_locale, revision_id=None):
                         pass
 
                 rev_form.save(doc)
+                # If this is an Ajax POST, then return a JsonResponse
+                if request.is_ajax():
+                    data = {
+                        'error': False,
+                        'new_revision_id': rev_form.instance.id,
+                    }
+
+                    return JsonResponse(data)
                 return redirect(doc)
+            else:
+                # If this is an Ajax POST, then return a JsonResponse with error
+                if request.is_ajax():
```

Comment:
This branch (failed AJAX edit of revision or combined translation form) is not covered by a test.